### DISCUSSION TITLE
COMP3438-SYSTEM PROGRAMMING-评论待收集

### DISCUSSION THREAD

**Discussion by WPCJATH** [ZHANG-CAIQI/PolyU_COMP#72] [课程评价]
评论请带上课时间(e.g., 21/22 sem1)或老师(e.g., Chen Dawen Peter, 陈大文, Peter)，谢谢！

**Comment by WANGaRuijie** [comment]
上这节课的老师是哪位？

**Reply by WPCJATH** [reply]
21 fall wqx, 22 fall wqx & SAXENA
